ui/lib/apps/Overview/components/RecentSlowQueries.tsx

```diff
@@ -19,14 +19,22 @@ export default function RecentSlowQueries() {
     queryTimeRange,
 
     errors,
-  } = useSlowQuery({ ...DEF_SLOW_QUERY_OPTIONS, limit: 10 }, false)
+
+    tableColumns,
+  } = useSlowQuery(
+    DEF_SLOW_QUERY_COLUMN_KEYS,
```

Comment:
The default columns to be shown in the Overview Page is very likely to be different from the columns to be shown in the SlowQueryList page, maybe better to split them like what you did to StatementTable.

Reply:
Currently they are exactly same, maybe we can change it when they start to be different.